taipy/core/notification/core_event_consumer.py

```diff
@@ -32,10 +32,18 @@ def process_event(self, event: Event):
             print(f"Received event created at : {event.creation_date}")
             pass
 
-    consumer = MyEventConsumer("consumer_1", event_queue)
+    registration_id, registered_queue = Notifier.unregister(
+        entity_type=EventEntityType.CYCLE,
+        entity_id="CYCLE_cycle_1",
+        operation=EventOperation.CREATION
+    )
+
+    consumer = MyEventConsumer("consumer_1", registered_queue)
     consumer.start()
     # ...
     consumer.stop()
+
+    Notifier.unregister(registration_id)
     ```
 
```

Comment:
After the example I would add a short paragraph to explain the code, and particularly
what are the events we are registering. We can also briefly explain the principle of the topic registration and redirect readers to the register function refman page to make it easier to understand how to build the topic.